commands.py

```diff
@@ -329,13 +347,21 @@ def repl_open(self, cmd_list, environment, syntax=None):
         if syntax is None:
             syntax = self.settings.get('repl_syntax')
 
+        syntaxpath = "Packages/%s.tmLanguage"
+        if syntax == "python":
+            syntaxpath = syntaxpath % "Python/Python"
+        elif syntax == "plaintext":
+            syntaxpath = syntaxpath % "Text/Plain text"
+        else:
+            raise ValueError("Unrecognized syntax setting '%s'" % (syntax,))
```

Comment:
We want to avoid raising an error at all costs. Instead of raising a `ValueError` we should just print something to debug output.